CharacterCreation/05_ArmourList.txt

```diff
@@ -597,7 +592,7 @@ Sec Module Hard Pts:	1
 Secondary Module Benefit: Provides +1 Cloak Turn
 Bonus, -1 Nanite Upkeep, per secondary module
 
-	=====   LEVEL 5 REQUIREMENT   =====
+=====   LEVEL 5 REQUIREMENT   =====
 	
 ---- Mk1 Passive Camouflage Module ----
   *Cloak Does Not Fail When Attacking*

```

Comment:
Is there a better way/place to put this warning?

Reply:
I'll put it in the desc

Reply:
Well, it looks pretty terrible... but Idk what to do with it so I made a single-point list.